File: app/src/main/java/com/slapshotapps/dragonshockey/models/PlayerGameStats.java
package com.slapshotapps.dragonshockey.models;

import android.support.annotation.Keep;

import java.util.ArrayList;

/**
 * Created on 11/5/16.
 */
@Keep
public class PlayerGameStats {

    public String playerStatsKey;
    public ArrayList<Player> players;
    public GameStats playerGameStats;

    public boolean isKeyValid() {
        return playerStatsKey != null;
    }

}

File: app/src/main/java/com/slapshotapps/dragonshockey/models/PlayerGameStats.kt
package com.slapshotapps.dragonshockey.models

import androidx.annotation.Keep
import java.util.*

/**
 * Created on 11/5/16.
 */
@Keep
class PlayerGameStats {

  var playerStatsKey: String? = null
  var players: ArrayList<Player>? = null
  var playerGameStats: GameStats? = null

  val isKeyValid: Boolean
    get() = playerStatsKey != null
}
